$MirOS: ports/print/cups/patches/patch-systemv_lp_c,v 1.2 2006/06/21 19:25:36 bsiegert Exp $
--- systemv/lp.c.orig	Tue Sep  5 22:45:24 2006
+++ systemv/lp.c	Wed Nov  8 18:01:54 2006
@@ -316,7 +316,7 @@ main(int  argc,				/* I - Number of comm
 	      num_copies = atoi(argv[i]);
 	    }
 
-            sprintf(buffer, "%d", num_copies);
+            snprintf(buffer, sizeof(buffer), "%d", num_copies);
             num_options = cupsAddOption("copies", buffer, num_options, &options);
 	    break;
 
